common/service/sensor/pdr.c

```diff
@@ -0,0 +1,53 @@
+#include <stdio.h>
+#include <string.h>
+#include "pdr.h"
+#include "sensor.h"
+#include "plat_pdr_table.h"
+#include "plat_sensor_table.h"
+#include "plat_ipmb.h"
+
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(pdr);
+
+extern PDR_numeric_sensor plat_pdr_table[];
+extern const int PDR_TABLE_SIZE;
+
+uint32_t pdr_count = 0;
```

Comment:
Can pdr_count and pdr_config_size have different values or always same?

Reply:
The pdr_config_size was planned to be another sensor config table size. Since we are not having sensor config table for PDR, I deleted it in this commit.